go.mod

```diff
@@ -2,32 +2,79 @@ module github.com/els0r/goProbe
 
 require (
 	github.com/els0r/status v1.0.0
+	github.com/fako1024/httpc v1.0.13
 	github.com/fako1024/slimcap v0.0.0-20230312144344-ffc7db30268a
+	github.com/gin-gonic/gin v1.9.0
 	github.com/go-chi/chi/v5 v5.0.7
 	github.com/json-iterator/go v1.1.12
 	github.com/spf13/cobra v1.5.0
+	github.com/spf13/viper v1.15.0
 	github.com/stretchr/testify v1.8.1
 	github.com/throttled/throttled v2.2.5+incompatible
 	github.com/zeebo/xxh3 v1.0.2
 	golang.org/x/exp v0.0.0-20230321023759-10a507213a29
 	golang.org/x/net v0.7.0
+	gopkg.in/yaml.v3 v3.0.1
 )
 
 require (
```

Comment:
Agreed with the first part. Should be as crisp as `go mod tidy`. The second part, I don't know how to check really. All I can tell you is that I tend to use frameworks too little vs. too much, so it's rather unlikely. 

Much unlike javascript, where half the internet broke because someone mucked up the `ToString()` method and that happened to be a third-party dependency...